Update demistoSDKWrapper.ts (#31)
demisto/vscode-extension: CHANGELOG.md

    Please use the **xsoar.demisto-sdk.Path** setting. The default behavior is to run all commands with `demisto-sdk <command>`. You may set the demisto-sdk executable with the given setting and use tools such as pipx.  
    Alternatively, you may also use it with your python executable of choice (`<python-path> -m demisto_sdk` as an example.)
- Added the **Run a Command in XSOAR** command.

> - Removed the support in the *maintenance* option of the *-u* flag in the **update-release-notes** command.

Reviewer: ```suggestion
- Removed the support of the **maintenance** option from the *Update Release Notes* command.
```
